packages/message/src/types.ts

```diff
@@ -1,15 +1,29 @@
-export type MessageTemplate = {
-  cumulus_meta: object
+import { Message } from '@cumulus/types';
+
+export interface MessageTemplateCumulusMeta {
+  queueExecutionLimits: {
+    [queueUrl: string]: number
+  }
+}
+
+export interface MessageTemplate {
```

Comment:
Consider `CumulusMessageTemplate` to help differentiate from something like an SQS message.